src/vs/workbench/browser/parts/editor/editorActions.ts

```diff
@@ -1553,3 +1553,78 @@ export class MoveEditorToNextGroupAction extends Action {
 		return TPromise.as(true);
 	}
 }
+
+export class MoveEditorToFirstGroupAction extends Action {
+
+	public static readonly ID = 'workbench.action.moveEditorToFirstGroupAction';
```

Comment:
@shobhitchittora remove the trailing `Action` from the action ids (e.g. `workbench.action.moveEditorToFirstGroup`)

Reply:
Done.